torchtitan/config_manager.py

```diff
@@ -332,8 +332,9 @@ def __init__(self):
             default="bfloat16",
             choices=["bfloat16", "float32"],
             help="""
-                torch dtype to use for parameters when applying mixed precision via FSDP.
-                This feature only takes effect when data_parallel_degree > 1
+                torch dtype to use for parameters when applying mixed precision.
+                When data_parallel_degree > 1, this changes FSDP's `param_dtype`.
```

Comment:
I think `data_parallel_degree` has been renamed to `data_parallel_shard_degree`.

Reply:
You are correct, apologies for not spotting the original error.